java/common/org.eclipse.theia.cloud.common/src/main/java/org/eclipse/theia/cloud/common/util/NamingUtil.java

```diff
@@ -129,45 +178,86 @@ public static String createName(Workspace workspace, String identifier) {
 	 * We fill remaining space with additional information about the workspace. This
 	 * may be trimmed away however.
 	 */
-	return createName(workspace.getMetadata().getUid(), identifier, getAdditionalInformation(workspace),
-		MAX_IDENTIFIER_LENGTH);
+	return createName("workspace", identifier, workspace.getSpec().getUser(),
```

Comment:
```suggestion
	return createName("ws", identifier, workspace.getSpec().getUser(),
```

Just as an idea, would `ws` be enough? We use it as a shortcut for `workspace` so it should be clear what is meant, and there is more space, for the user and appDef name?

Reply:
You are right. I thought `workspace` is more readable. However someone accessing the resources with kubectl is probably knowledgable enough to understand the ws- prefix, too